web-registry/src/lib/wallet/wallet.constants.ts

```diff
@@ -0,0 +1,16 @@
+export const AUTO_CONNECT_WALLET_KEY = 'auto_connect_wallet';
+export const WALLET_CONNECT_KEY = 'walletconnect';
+export const WALLET_CONNECT_BRIDGE_URL = 'https://bridge.walletconnect.org';
+export const WALLET_CONNECT_SIGNING_METHODS = [
+  'keplr_enable_wallet_connect_v1',
+  'keplr_sign_amino_wallet_connect_v1',
+];
+
+export const walletConnectClientMeta = {
+  name: 'Regen Network Registry',
```

Comment:
```suggestion
  name: 'Regen Marketplace',
```
We are using "marketplace" instead of "registry" now